index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+
+    <title></title>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+    <!--script src='main.js'></script-->
+</head>
+<body>
+    <nav class="toolbar">
+        <img id="img1" src="image_geometry_1.svg" alt="">
+        <div class="ic_icon">
+          <button id="btn"></button>
+          <button id="btn"></button>
+          <button id="btn"></button>
+        </div>
+    </nav>
+    <section class="headline">
+        <div class="group48">
+            <p id="title"><h1> CharlieGiorgania</h1></p>
```

Comment:
As requested by the previous reviewer, The main title i.e. `CharlieGiorgani`  should be a `h1` tag.

Standalone `h1` not inside a `p` tag ☺
```HTML
<h1>CharlieGiorgania</h1>
```

If you want that google (and other search engines) to know about a specific topic, we use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. 🤓